serverless/serverless-overview.adoc

```diff
@@ -0,0 +1,27 @@
+[id="serverless-overview"]
+= Red Hat OpenShift Serverless overview
```

Comment:
Is this the overview or the architecture guide?

Are you planning to add more modules to this assembly? If it's complete, I think that you need to retitle the assembly to say that it's the arch guide and adjust the module's initial headings.

Reply:
Separated these - I was inc. the architecture in this overview but it's out now.